pages/clustering/high-availability.mdx

```diff
@@ -630,9 +627,14 @@ expiration is set to 2000ms so that cluster can never get into situation where m
 the ability to survive occasional network hiccups without triggering leadership changes.
 
 
+## Data center failure
+
+The architecture we currently use allows us to deploy coordinators in 3 data centers and hence tolerate a failure of the whole data center. Data instances can be freely
+distributed in any way you want between data centers.
```

Comment:
I would extend this with some notes on the expected system requirements, e.g., the latency should be under N ms 🤔

Reply:
I think it's not necessary. Failover will be slower but slower network IMO still doesn't disqualify the architecture.